core/main/src/state/cap/permitted_state.rs

```diff
@@ -205,31 +202,9 @@ impl PermissionHandler {
         }
     }
 
-    pub async fn device_fetch_and_store(state: &PlatformState, app_id: &str) -> RippleResponse {
-        let mut client = state.get_client().get_extn_client();
-        let resp = client
-            .request(AppsRequest::GetFireboltPermissions(app_id.to_string()))
-            .await?;
-
-        let mut permissions = match resp.payload {
-            ExtnPayload::Response(response) => match response {
-                ExtnResponse::Permission(perms) => perms,
-                ExtnResponse::Error(e) => {
-                    error!("device_fetch_and_store: e={:?}", e);
-                    return Err(e);
-                }
-                _ => {
-                    error!("device_fetch_and_store: Unexpected response");
-                    return Err(RippleError::ExtnError);
-                }
-            },
-            _ => {
-                error!("device_fetch_and_store: Unexpected payload");
-                return Err(RippleError::ExtnError);
-            }
-        };
-
-        Self::process_permissions(state, app_id, &mut permissions)
+    pub async fn device_fetch_and_store(_state: &PlatformState, _app_id: &str) -> RippleResponse {
+        error!("device_fetch_and_store: Not supported");
+        return Err(RippleError::NotAvailable);
```

Comment:
When the new PackageManager is available and if it provides app capabilities then this will fetch those permissions similar to before. Right now we're always going to the cloud for permissions.